imap-types/src/rfc3501/command.rs

```diff
@@ -1178,6 +1182,84 @@ pub enum CommandBody<'a> {
 
     #[cfg(feature = "ext_compress")]
     Compress { algorithm: CompressionAlgorithm },
+
+    /// [4.1. SETQUOTA Command](https://www.rfc-editor.org/rfc/rfc2087#section-4.1)
+    /// Arguments:  quota root
+    ///             list of resource limits
+    ///
+    /// Data:       untagged responses: QUOTA
+    ///
+    /// Result:     OK - setquota completed
+    ///             NO - setquota error: can't set that data
+    ///             BAD - command unknown or arguments invalid
+    ///
+    /// The SETQUOTA command takes the name of a mailbox quota root and a
+    /// list of resource limits. The resource limits for the named quota root
+    /// are changed to be the specified limits.  Any previous resource limits
+    /// for the named quota root are discarded.
+    ///
+    /// If the named quota root did not previously exist, an implementation
+    /// may optionally create it and change the quota roots for any number of
+    /// existing mailboxes in an implementation-defined manner.
+    #[cfg(feature = "ext_quota")]
+    SetQuota {
+        /// The name of a mailbox quota root.
+        quota_root: Text<'a>,
```

Comment:
I'm not sure `Text` is the right type to use here. If I'm not mistaken, RFC 9208 defines it as ...

```abnf
setquota = "SETQUOTA" SP quota-root-name SP setquota-list
quota-root-name = astring
setquota-list = "(" [setquota-resource *(SP setquota-resource)]")"
```

Thus, `quota_root` should be an `AString`.